### ISSUE TITLE
App migrated from v4 to v5 ignores configuration

### ISSUE THREAD

**Issue by sdobber** [GenieFramework/Genie.jl#550]
**Describe the bug**
I migrated two different Apps from Genie v4 to v5. When starting the migrated app in production mode with `up()`, the `server_host` and `server_port` settings from `/config/env/prod.jl` are ignored, and the server starts with IP `127.0.0.1`. I noticed that the file content changed when comparing with the output of `Genie.Generator.newapp(...)`. But even changing the `prod.jl` file from 
```julia
using Genie.Configuration, Logging

const config = Settings(
  server_port=8000,
  server_host="192.168.0.200",
  log_level=Logging.Info,
  log_to_file=true,
  server_handle_static_files=true
)

ENV["JULIA_REVISE"] = "off"
```
to
```julia
using Genie, Logging

Genie.Configuration.config!(
  server_port=8000,
  server_host="192.168.0.200",
  log_level=Logging.Info,
  log_to_file=true,
  server_handle_static_files=true,
  path_build                      = "build",
  format_julia_builds             = false,
  format_html_output              = false
)

ENV["JULIA_REVISE"] = "off"
```
does not allow to start the server with the desired IP.

**Error stacktrace**
No error is thrown, just the normal message
```
julia> up()
┌ Info: 2022-07-28 09:48:38 
└ Web Server starting at http://127.0.0.1:8000 
Genie.Server.ServersCollection(Task (runnable) @0x0000000177b0cd30, nothing)
```
where the IP is not the one from the configuration

**To reproduce**
I have seen this 2 times when migrating an app. I am not sure if the original Genie version that was used for bootstrapping the app plays a role - it might well be that it was v3 or earlier.

**Expected behavior**
Genie respects the settings defined in the file 😁 

**Additional context**
Please include the output of 
```
julia> versioninfo()
Julia Version 1.7.3
Commit 742b9abb4d (2022-05-06 12:58 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin21.4.0)
  CPU: Apple M1 Pro
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-12.0.1 (ORCJIT, westmere)
Environment:
  JULIA_EDITOR = code
  JULIA_NUM_THREADS = 4
  JULIA_REVISE = off
```
and
```
pkg> st
     Project DanceItServer v0.1.0
      Status `~/Uni/Code/DanceIt/DanceItServer/Project.toml`
  [1520ce14] AbstractTrees v0.4.2
  [8aed41a2] BallroomSkatingSystem v0.1.0
  [ad839575] Blink v0.12.5
  [336ed68f] CSV v0.10.4
  [aaaa29a8] Clustering v0.14.2
  [a93c6f00] DataFrames v1.3.4
  [c43c736e] Genie v5.0.0
  [23fbe1c1] Latexify v0.15.16
  [e6f89c97] LoggingExtras v0.4.9
  [739be429] MbedTLS v1.1.1
  [bac558e1] OrderedCollections v1.4.1
  [f065f642] Publish v0.10.0
  [6cfbe66a] QRCode v0.2.0
  [295af30f] Revise v3.3.3
  [0aa819cd] SQLite v1.4.1
  [340e8cb6] SearchLight v2.4.1
  [21a827c4] SearchLightSQLite v2.2.0
  [88034a9c] StringDistances v0.11.2
  [fdbf4ff8] XLSX v0.7.10
  [a5390f91] ZipFile v0.9.4
  [ade2ca70] Dates
  [56ddb016] Logging
  [44cfe95a] Pkg
  [9a3f8284] Random
  [ea8e919c] SHA
```

**Comment by sdobber** [comment]
Maybe this is not a migration issue after all. When creating a new app with `Genie.Generator.newapp("Name")`, changing `server_host` in the `prod.jl` file and starting everything in a new session with
```julia
 ENV["GENIE_ENV"] = "prod"
 include("bootstrap.jl")
```
then the `server_host`-entry in ` UserApp.Genie.config` points to 127.0.0.1 and not to the value from `prod.jl`.

**Comment by essenciary** [comment]
Thanks, I'll investigate

**Comment by essenciary** [comment]
@sdobber Should be fixed now in Genie 5.1.1
